### ISSUE TITLE
Listing Droplets fail

### ISSUE THREAD

**Issue by iammilton82** [phillbaker/digitalocean-node#5]
Whenever I initiate the method for listing droplets, it hangs.

Here's the code that I'm using:

`
client.droplets.list(function(err, data){

```
        data._digitalOcean.body = {};

        if(err){

            res.json({
                status: 400,
                operation: 201,
                message: "failure",
                data: data
            });

        } else {

            data._digitalOcean.body = {};

            res.json({
                status: 200,
                operation: 200,
                message: "Success",
                data: data
            });
        }

    });`
```

**Comment by phillbaker** [comment]
Hi @iammilton82, what version of the client are you using? And what version of request do you have installed?

**Comment by iammilton82** [comment]
@phillbaker using version 0.8.0 of the client. 
Version 2.74.0 of request.

**Comment by phillbaker** [comment]
@iammilton82 this should be fixed in 0.8.1, thanks for reporting the bug. It looks like this was introduced in 0.7.1, so 0.7.1, 0.7.2, 0.7.3, 0.7.5 and 0.8.0 are affected.

**Comment by iammilton82** [comment]
Awesome. thanks!
